DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/src/CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ set(EMULATOR_LINK_FLAGS "-fintelfpga ${AC_TYPES_FLAG} ${ENABLE_USM}")
 set(SIMULATOR_COMPILE_FLAGS "-fintelfpga -Wall -fbracket-depth=512 ${AC_TYPES_FLAG} ${ENABLE_USM} ${SENSOR_SIZE_FLAG} ${NUM_SENSORS_FLAG} ${QRD_MIN_ITERATIONS_FLAG} ${STREAMING_PIPE_WIDTH_FLAG}")
```

Comment:
Seems a bit odd that the sim flow doesn't specify a target...

Reply:
Looks like a bug - not my code but I'll fix it here.